apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useDevice.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from "react";
+import DeviceInfo from "react-native-device-info";
+
+export function useDevice() {
+  const [device, setDevice] = useState<string>();
+  useEffect(() => {
+    async function getDeviceInfo() {
+      const deviceInfo = await DeviceInfo.getDeviceName();
```

Comment:
see comment on slack. i think we need not to use device name, but instead figure out a minimal way to get things like `Android 18`, `iPhone 12`